archinstall/lib/disk/device_handler.py

```diff
@@ -422,11 +422,18 @@ def _lvm_info_with_retry(
 		cmd: str,
 		info_type: Literal['lv', 'vg', 'pvseg'],
 	) -> LvmVolumeInfo | LvmGroupInfo | LvmPVInfo | None:
-		while True:
+		# Retry for up to 30 seconds (10 attempts * 3 seconds)
```

Comment:
If it's a large disk this may not be enough, it should be higher than that maybe 5min?